gitensite/apps/content/views.py

```diff
@@ -23,6 +23,13 @@ def get_template_names(self, **kwargs):
                 issue=str(self.kwargs['issue'])
                 )]
 
+class UpdatesView(TemplateView):
```

Comment:
When all I'm doing is setting a template, I usually set it in urls.py:

   `url(r'^updates/?$', TemplateView.as_view(template_name="privacy.html"), name='updates'),`